DiskImage.cpp

```diff
@@ -1394,7 +1394,7 @@ void TDiskImage::readFDI(int hfile, bool readonly)
 		{
 			if (rsize < fdiOFF)
 			{
-				delete tracksinfo;
```

Comment:
This is a memory leak since the pointer was allocated as an array but deleted as a pointer (only the first element will be deleted, the rest will be leaked).